domain/src/main/java/org/woowatechcamp/banchan/domain/usecase/BestUseCaseImpl.kt

```diff
@@ -0,0 +1,12 @@
+package org.woowatechcamp.banchan.domain.usecase
+
+import org.woowatechcamp.banchan.domain.entity.CategoryInfo
+import org.woowatechcamp.banchan.domain.repository.HomeRepository
+import javax.inject.Inject
+
+class BestUseCaseImpl @Inject constructor(
+    private val homeRepository: HomeRepository,
+) : BestUseCase {
+    override suspend fun invoke(): Result<List<CategoryInfo>> =
+        kotlin.runCatching { homeRepository.getCategory() }
```

Comment:
```suggestion
            runCatching { homeRepository.getCategory() }
```
kotlin이라고 적힌 부분은 제거해도 괜찮아요!

Reply:
자동 생성 코드로 작성했던 부분인데 수정하도록 하겠습니다 ㅋㅋ